src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -130,17 +143,22 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
 
     ### `route`
 
-    For `route`, it will use the first part of the class name as the resource name,
+    [The macros `route` and `nested-route` do still exist, and automatically imply the default route paths,
+    however, their depreciation is [discussed](https://github.com/luckyframework/lucky/issues/789). Moving
```

Comment:
I believe this should be `deprecation`

Reply:
Um no, that sounds rather hard to me. Think `depreciated` is common for phasing out software functions.

Reply:
Depreciation is for finance assets https://www.investopedia.com/terms/d/depreciation.asp 

Deprecation is for software https://techterms.com/definition/deprecated so I think we should be use `deprecation`

Reply:
Ok, distinguished language, seems to be depend on the project.